index.html

```diff
@@ -0,0 +1,198 @@
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <title>portofolio</title>
+</head>
+<body>
+    
+    <div class="container">
+        <div class="card">
+        <header>
+            <div class="fcontainer">
+                <a class="logo">Saba</a>
+                <nav id="items">
+                  <i class="fa fa-bars"></i>
+                </nav>
+              </div>
+          </header>
+           <div class="fcontent">
+            <h1 class="name">I'M Saba Ahmad <br> Glad to see <br> you!</h1>
+            <p class="paragraph1">
+              I'm a software developer! I can help you  build a product , feature or website look  through some of my work  experience ! if you like what you see and  have a project you need coded, don't hesitate to contact me
+          </p>
+           </div>
+           <div class="connect">
+            <h3>LET'S  CONNECT</h3>
+        </div>
+        <div class="icons">
+            <ul> 
+           <li><img src="image/image8.png" alt="twitter"></li> 
+          <li><img src="image/imge5.png" alt="twitter"></li> 
+          <li><img src="image/image9.png" alt="twitter"></li> 
+          <li><img src="image/image6.png" alt="twitter"></li>
+          <li><img src="image/image7.png" alt="twitter"></li> 
+           </ul>
+            </div>
+</div>
+        
+        <section>
+            <div class="card1">
+            <img src="image/image2.png" alt="hello">
+        <h1>TONIC</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+    </div>
+
+        <div class="card2">
+            <img src="image/card2.png" alt="some">
+        <h1>Multi-Post <br>Stories</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+         </div>
+
+
+        <div class="card3">
+            <img src="image/card3.png" alt="">
+        <h1>TONIC</h1> 
```

Comment:
Please note that there can only be one `<h1>` element in your page. I suggest you use any other header tags `<hx>`
if you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.